src/runtime/node/console/index.ts

```diff
@@ -62,4 +68,10 @@ export default <typeof console>{
   timeStamp,
   trace,
   warn,
-};
+} satisfies typeof console;
+
+export { default as _times } from "../../mock/proxy";
+export {
+  default as _stdoutErrorHandler,
+  default as _stderrErrorHandler,
+} from "../../mock/noop";
```

Comment:
Code style: we usually have default export at last

Reply:
this is eslint rewriting my code in a way I don't like. should I disable it?

Reply:
It shouldn't order let me check locally 👍🏼

Reply:
I think the issue was that I imported `noop` once and then exported it twice - eslint combined the two into a single export, and along the way changed the order. since you kept it as a single export, you were able to move it up.